.release/linux/package/etc/boundary.d/boundary.hcl

```diff
@@ -1,3 +1,112 @@
+# # Note that this is an example systemd file and is not intended to be functional as-is.
```

Comment:
This isn't really a systemd-managed file; it's just a config file mentioned by a systemd unit.